kong/plugins/acme/handler.lua

```diff
@@ -221,6 +221,12 @@ function ACMEHandler:access(conf)
     end
 
     if captures then
+      -- creating account through proxy side with "kong" storage in Hybrid mode is not supported
+      -- if this is just a sanity test, we always return 404 status
+      if captures[1] == "x" and kong.configuration.role == "data_plane" and conf.storage == "kong" then
+        return kong.response.exit(404, "Not found\n")
```

Comment:
Actually, we do not need to include a reason phrase here(it's added automatically) and the second arg is the response body.
```suggestion
        return kong.response.exit(404)
```